### PR TITLE
Feature/sw-5-02/47-administrate-childrens-information

### LINKED ISSUE #47: As a admin i want to be administrate childrens information

### PR DESCRIPTION
**Pull request by JacobInberg**
# Description

*Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.*

Fixes #47 

## Type of change
*Delete unchecked boxes (only for Type of change)*

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Acceptence Test

*Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration*

**Development Configuration**
*Type "flutter --version" and "dart --version" in your CMD to check versions.*

* Flutter version:
* Dart version:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas, if necessary
- [ ] I have made corresponding changes to the documentation, if necessary
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works, if necessary
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have Acceptance Tested this on an iOS device
- [ ] I have Acceptance Tested this on an Android device

### CODE REVIEW

File: foodplanner/lib/models/user.dart [hunk 1]
```diff
@@ -4,30 +4,34 @@ class User {
...
+    required this.archived,
   });
 
   factory User.fromJson(Map<String, dynamic> json) {
     return switch (json) {
       {
         'id': int id,
-        'firstName': String firstName,
-        'lastName': String lastName,
+        'first_name': String firstName,
```
Comment (DollyTheSau) [inline foodplanner/lib/models/user.dart]: Hvorfor går vi fra camelCase til snake case?

File: foodplanner/lib/pages/administrate_children.dart [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                ),
+              ),
+            ).then((_) {
+              fetchChildren();
+            });
+          },
+        ),
+      ],
+    );
+  }
```
Comment (DollyTheSau) [inline foodplanner/lib/pages/administrate_children.dart:104]: hvorfor bruger vi ikke custombutton?

File: foodplanner/lib/pages/settings/settings.dart [hunk 3]
```diff
@@ -105,6 +106,12 @@ class _SettingsPage extends State<Settings> {
               SizedBox(width: 10),
             ],
           ),
+          'ctaFunction': () {
+            Navigator.push(
+              context,
+              MaterialPageRoute(builder: (context) => AdministrateChildren()),
+            );
```
Comment (DollyTheSau) [inline foodplanner/lib/pages/settings/settings.dart:120]: tage op til næste møde: skifte fra navigator.push til GoRouter så vi ikke laver layered widgets og vi kan få security indover. Navigator giver god mening når vi vil have et popup.

File: foodplanner/lib/services/fetch_auth.dart [hunk 4]
```diff
@@ -32,7 +32,6 @@ class AuthService {
         return error;
         //throw Exception('Failed to load auth data');
       }
-      // sabrina carpenter <3
```
Comment (DollyTheSau) [inline foodplanner/lib/services/fetch_auth.dart]: ommer

**Review by DollyTheSau — APPROVED** [review]
Overall ville det være nice hvis de pages I har lavet også bliver tilføjet til routes og index så det er nemmere at fikse når vi begynder med GoRouter på alt. Det gør koden meget mere læselig også. 

Ellers fint <3

File: foodplanner/lib/models/user.dart (hunk 1 above)
Comment (JacobInberg) [reply foodplanner/lib/models/user.dart]: Tror ikke den her har været brugt før vi gjorde det, men den har altid være blevet sendt som snakecase i fra backenden no?

File: foodplanner/lib/pages/administrate_children.dart (hunk 2 above)
Comment (JacobInberg) [reply foodplanner/lib/pages/administrate_children.dart:104]: I have exaplained yes?

File: foodplanner/lib/pages/settings/settings.dart (hunk 3 above)
Comment (JacobInberg) [reply foodplanner/lib/pages/settings/settings.dart:120]: Aight men er den her så resolved eller?

File: foodplanner/lib/pages/administrate_children.dart (hunk 2 above)
Comment (DollyTheSau) [reply foodplanner/lib/pages/administrate_children.dart:104]: yes

File: foodplanner/lib/models/user.dart (hunk 1 above)
Comment (DollyTheSau) [reply foodplanner/lib/models/user.dart]: idk men I gik bare den anden vej rundt lige ovenover 😐

File: foodplanner/lib/pages/settings/settings.dart (hunk 3 above)
Comment (DollyTheSau) [reply foodplanner/lib/pages/settings/settings.dart:120]: for now yes

**Pull request merged**